.travis.yml

```diff
@@ -54,6 +54,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - "pip freeze"
```

Comment:
**Comment**: Are we doing this in all repos now? I think that by expanding the [pip install section](https://travis-ci.org/github/inveniosoftware/invenio-admin/jobs/682936264#L370) we should be able to see which exact dependencies were installed.

Reply:
I personally was not able to do a "diff" with the other two sections expanded. There is too much content that changes from one version to the other. The section you mention is okay to look for a module (if you know what you are looking for).

However, in this case was only failing in "lowest" and I didn't know which modules change. By just diffing the output of freeze I spotted the modules in one glance. I added it because I have seen it in other modules.

Do you think is unnecessary or costly? or it would be to just avoid "duplication" of output?

Reply:
No, I was just wondering why we were not doing it before, now I know something new 😊.